src/highrise/models.py

```diff
@@ -368,9 +368,14 @@ class SessionMetadata:
     Initial session data.
 
     This will be sent once, as the first message when a connection is established.
+    user_id is the bot's user id.
+
+
     """
 
     user_id: str
+    rate_limits: dict[str, tuple[int, float]]
+    sdk_versions: dict[str, str] | None = None
```

Comment:
Can this just be a `str | None`? None if no recommendation, and a version string if the user is recommended to upgrade?

Reply:
Done